src/google/protobuf/descriptor.ts

```diff
@@ -1810,100 +2002,116 @@ export const FieldDescriptorProto = {
     if (message.name !== "") {
       writer.uint32(10).string(message.name);
     }
+
     if (message.number !== 0) {
       writer.uint32(24).int32(message.number);
     }
-    if (message.label !== 1) {
+
+    if (message.label !== 0) {
       writer.uint32(32).int32(message.label);
     }
-    if (message.type !== 1) {
+
+    if (message.type !== 0) {
       writer.uint32(40).int32(message.type);
     }
+
     if (message.typeName !== "") {
       writer.uint32(50).string(message.typeName);
     }
+
     if (message.extendee !== "") {
       writer.uint32(18).string(message.extendee);
     }
+
     if (message.defaultValue !== "") {
       writer.uint32(58).string(message.defaultValue);
     }
+
     if (message.oneofIndex !== 0) {
       writer.uint32(72).int32(message.oneofIndex);
     }
+
     if (message.jsonName !== "") {
       writer.uint32(82).string(message.jsonName);
     }
+
     if (message.options !== undefined) {
       FieldOptions.encode(message.options, writer.uint32(66).fork()).ldelim();
     }
-    if (message.proto3Optional === true) {
-      writer.uint32(136).bool(message.proto3Optional);
-    }
```

Comment:
this may be due to a different descriptor.proto that we've embedded. However, I really don't think this matters.